docs/en_US/network-dumps.md

```diff
@@ -42,7 +42,7 @@ The easiest way to dump game server packets from a 3DS is to use [HokakuCTR](htt
 
 To use HokakuCTR, ensure you are updated to the latest Luma release. The latest Luma release now supports 3GX plugins. Download the [latest release](https://github.com/PretendoNetwork/HokakuCTR/releases/latest). Place this file on the SD card at `luma/plugins` and rename it `default.3gx`.
 
-When launching a game you will see 1 of 2 notifications on screen. You will either see `Not Ready` or `Ready`. If you see `Not Ready`, your game is not compatible. If you see `Ready`, your game is compatible but has not necessarily started sending data. If you see nothing, either your game is not compatible or the plugin is not installed. If your game is compatible, it will start dumping game traffic to a PCAP file once the game has connected to the server and begun sending data. You will know the game has connected to the server when you see either `Detected NEX buffer type: V0` or `Detected NEX buffer type: V1` on screen. These notifications indicate that the game has begun sending data. If you do not see one of these notifications, no data will be in your dump.
+When launching a game you will see 1 of 2 notifications on screen. You will either see `Not Ready` or `Ready`. If you see `Not Ready`, your game is not compatible. If you see `Ready`, your game is compatible but has not necessarily started sending data. If you see nothing, either your game is not compatible, the plugin loader is disabled or the plugin is not installed. You can check if the plugin loader is disabled by opening the Rosalina menu (L + D-Pad down + Select) and ensure "Plugin Loader" is enabled. If your game is compatible, it will start dumping game traffic to a PCAP file once the game has connected to the server and begun sending data. You will know the game has connected to the server when you see either `Detected NEX buffer type: V0` or `Detected NEX buffer type: V1` on screen. These notifications indicate that the game has begun sending data. If you do not see one of these notifications, no data will be in your dump.
```

Comment:
only a small nitpick from me

```suggestion
When launching a game you will see 1 of 2 notifications on screen. You will either see `Not Ready` or `Ready`. If you see `Not Ready`, your game is not compatible. If you see `Ready`, your game is compatible but has not necessarily started sending data. If you see nothing, either your game is not compatible, the plugin loader is disabled or the plugin is not installed. You can check if the plugin loader is disabled by opening the Rosalina menu (L + D-Pad down + Select) and ensure the "Plugin Loader" option is enabled. If your game is compatible, it will start dumping game traffic to a PCAP file once the game has connected to the server and begun sending data. You will know the game has connected to the server when you see either `Detected NEX buffer type: V0` or `Detected NEX buffer type: V1` on screen. These notifications indicate that the game has begun sending data. If you do not see one of these notifications, no data will be in your dump.
```